services/api/events.js

```diff
@@ -16,22 +17,32 @@ const eventSchema = Schema({
     date: Date,
     type: String
 }, { collection: 'events' })
-const eventModel = conn.model('events', eventSchema)
 
+const eventModel = conn.model('events', eventSchema)
 
 function searchEvents(req, res, next) {
     let query = {}
     let body = req.body
 
     // TODO(Ray): there will be a validator being reponsible for it
```

Comment:
Needs a validator to valid:
1) the structure of "timestamp" object in req.body, does it always have "start" and "end" both be presented, or there is situation that only one of them be presented? 
2) the type of "start" and "end"

Reply:
1. yes, I prefer there is situation that only one of them be presented. I will modify code.
2. I don't understand what u mean?